src/ring_theory/adjoin/basic.lean

```diff
@@ -49,6 +49,14 @@ algebra.gc _ _
 theorem adjoin_mono (H : s ⊆ t) : adjoin R s ≤ adjoin R t :=
 algebra.gc.monotone_l H
 
+lemma coe_inf (S T : subalgebra R A) : (↑(S ⊓ T) : set A) = (S : set A) ∩ (T : set A) :=
+begin
+  apply le_antisymm,
+  { simp },
+  { rw ←galois_insertion.l_inf_u (@algebra.gi R A _ _ _),
+    exact algebra.subset_adjoin }
+end
```

Comment:
I'm slightly surprised this isn't true by refl, but I guess we constructed the lattice differently on algebra compared to how we do for other types.

Reply:
For those interested the relevant Zulip thread is https://leanprover.zulipchat.com/#narrow/stream/113489-new-members/topic/Intersection.20of.20subalgebras

I will try to modify the definition in another PR.